tests/manager/manager-ctor-write_concern-error-001.phpt

```diff
@@ -1,66 +1,31 @@
 --TEST--
-MongoDB\Driver\Manager::__construct(): invalid write concern
+MongoDB\Driver\Manager::__construct(): invalid write concern (w)
 --FILE--
 <?php
 
 require_once __DIR__ . '/../utils/tools.php';
 
-echo throws(function() {
-    $manager = new MongoDB\Driver\Manager('mongodb://127.0.0.1/?w=-1&journal=true');
-}, "MongoDB\Driver\Exception\InvalidArgumentException"), "\n";
-
-echo throws(function() {
-    $manager = new MongoDB\Driver\Manager('mongodb://127.0.0.1/?w=0&journal=true');
-}, "MongoDB\Driver\Exception\InvalidArgumentException"), "\n";
-
-echo throws(function() {
-    $manager = new MongoDB\Driver\Manager('mongodb://127.0.0.1/?w=-1', ['journal' => true]);
-}, "MongoDB\Driver\Exception\InvalidArgumentException"), "\n";
-
-echo throws(function() {
-    $manager = new MongoDB\Driver\Manager('mongodb://127.0.0.1/?w=0', ['journal' => true]);
-}, "MongoDB\Driver\Exception\InvalidArgumentException"), "\n";
-
-echo throws(function() {
-    $manager = new MongoDB\Driver\Manager('mongodb://127.0.0.1/?journal=true', ['w' => -1]);
-}, "MongoDB\Driver\Exception\InvalidArgumentException"), "\n";
```

Comment:
The write concern URI error tests are now organized by the option (like read preferences). All of these "journal" test cases were moved to `manager-ctor-write_concern-error-005.phpt` (git considers it a copy of `manager-ctor-write_concern-error-001.phpt` with some modifications).